src/index.tsx

```diff
@@ -396,11 +396,11 @@ class App extends React.Component<{}, AppState> {
 
     if (event.key === "Escape") {
       clearSelection();
-      drawScene();
+      this.forceUpdate();
```

Comment:
All the `drawScene` have been updated with `this.forceUpdate()`. The next step is going to use this.setState instead of forceUpdate